src/styles/preferences/index.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+
+import Button from '../../components/common/button';
+
+const Section = styled.section`
+  width: 100%;
+  padding: 64px 16px;
+  max-width: 1200px;
+  margin: auto;
+`;
+
+const Title = styled.header`
+  font-size: ${props => props.theme['font-2xl']};
+  font-weight: ${props => props.theme['font-bold']};
+  text-align: center;
+  margin-bottom: 24px;
+
+  ${props => props.theme.mq.tablet} {
+    font-size: ${props => props.theme['font-xl']};
+  }
+`;
+
+const CardContainer = styled.article<{ recommend?: boolean }>`
+  display: grid;
+  grid-gap: 24px;
+  width: 100%;
+  grid-template-columns: ${props => (props.recommend ? `1fr 1fr` : `1fr 1fr 1fr`)};
+  justify-items: center;
+
+  ${props => props.theme.mq.tablet} {
+    grid-template-columns: 1fr 1fr;
+  }
+`;
+
+const CompletedButton = styled(Button)`
+  font-weight: ${props => props.theme['font-semibold']};
+  color: white;
+  background-color: ${props => props.theme['mainColor']};
+  margin-bottom: 18px;
+  float: right;
+`;
+
+const ButtonText = styled.span``;
+
+const LoadingContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #9e9ec7;
+  height: 100vh;
```

Comment:
화면이 작아질수록 아래 공백이 남아서 `100vh`로 처리해놓았는데 혹시 다른 방법이 있을까요 .. 🤔 !?

Reply:
100vh에서 헤더(75px) -푸터 (225px) 빼주면 헤더와 푸터 크기 합쳐서 화면높이가 됩니다

```suggestion
  height: calc(100vh - 300px);
```